sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 
 ### Other Changes
 
+## 5.1.1 (2021-11-19)
+### Breaking changes
+- Enum `EntityCategory.IPAddress` now uses the underlying string `IPAddress` value instead of `IP` to align with the Text Analytics service behavior.
+
+### Bug Fixes
+- Long-Running operation rehydration has been patched to stop throwing a `NullReferenceException`. Issue [24692](https://github.com/Azure/azure-sdk-for-net/issues/24692).
+
```

Comment:
Wondering if we should include this set of changes in 5.2.0-beta.3 too. Users who update from 5.2.0-beta.2 to 5.2.0-beta.3 could miss it from the changelog.

Reply:
I agree that it would be a good idea;  more visibility is a good thing!